UnboundLib/Networking/PingMonitor.cs

```diff
@@ -74,7 +63,7 @@ public override void OnJoinedRoom()
             }
 
             // Run an RPC after a half second, to give the client time to connect to the lobby completely.
-            this.ExecuteAfterSeconds(0.5f, () =>
+            this.ExecuteAfterFrames(15, () =>
```

Comment:
I agree with @willis81808 on this. Any delay that's in frames instead of realtime should have a good justification for being such.

Reply:
I mean from my point of view this is exactly how it should be as we are relying only on the few things to load that are frame specific for the lobby. Though if you think it's necessary to wait that delay for something can you please explain why exactly?

Reply:
Based on the comment above that line, it’s not waiting on anything frame dependent, but rather network dependent. And that correlates better with real-time delays, rather than frame delays (which may not give enough time for a reply or whatever on a slow network)

Reply:
I could have mistaken it for something else, but from what I understand we already joined the room, the stuff that's being setup is on the client after this and just sends an rpc later to start updating the pings, it's nothing related to actually assuring lobby connection

Reply:
I didn't make the PingMonitor so I can't say why the creator felt it was necessary to wait a half second in realtime before updating pings across clients. Can you explain why it is frame dependent, and why 15 is a good number?